hpx/util/bind_front.hpp

```diff
@@ -13,6 +13,7 @@
 #include <hpx/util/decay.hpp>
 #include <hpx/util/detail/pack.hpp>
 #include <hpx/util/invoke.hpp>
+#include <hpx/util/one_shot.hpp>
```

Comment:
Why include this here? It isn't used in the edited version as far as I can see.

Reply:
Looks like a leftover of the refactoring, same for `bind_back.hpp`